test/tf/hcloud/versions.tf

```diff
@@ -0,0 +1,9 @@
+terraform {
+  required_providers {
+    hcloud = {
+      source  = "terraform-providers/hcloud"
+      version = "1.21.0"
+    }
+  }
+  required_version = ">= 0.13"
```

Comment:
Right now our circle ci configuration is setup to use terraform 0.12. So I have a feeling this will fail.

@alvin-huang Do you think we should update all the tf in this repo to 0.13 and use it or should we roll the version requirements of this PR back to tf 0.12 and do the 0.13 upgrade later.

Reply:
I think it would be best to pin it back to 0.12 and we can go through and migrate all the providers to 0.13 config at once at a separate time. @shortner can you make that change?

Reply:
Friendly ping @shortner. :)